lib/ddtrace/ext/ci.rb

```diff
@@ -63,24 +63,32 @@ def filter_sensitive_info(url)
 
       def extract_appveyor(env)
         url = "https://ci.appveyor.com/project/#{env['APPVEYOR_REPO_NAME']}/builds/#{env['APPVEYOR_BUILD_ID']}"
+
+        if !env['APPVEYOR_REPO_PROVIDER'].nil? && env['APPVEYOR_REPO_PROVIDER'] == 'github'
```

Comment:
There's no need to nil check here, as `nil == 'github'` returns false:
```suggestion
        if env['APPVEYOR_REPO_PROVIDER'] == 'github'
```